content/en/platform/corda/5.0/developing-applications/getting-started/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -0,0 +1,349 @@
+---
+date: '2023-06-01'
+title: "Ledger Extensions API Reference"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-utxo-ledger-extensions-api-reference
+    parent: corda5-utxo-advanced-ledger-extensions
+    weight: 4600
+section_menu: corda5
+---
+
+# Advanced Contract Design
```

Comment:
```suggestion
# Ledger Extensions API Reference
```

Reply:
Have updated the headings based on Christian's suggestion to read:

# Ledger Extensions API Reference

## Advanced Contract Design

...
## Advanced Ledger Types

### Base API